### ISSUE TITLE
python 2.7 ?

### ISSUE THREAD

**Issue by vsraptor** [JoaoFelipe/apted#2]
Does it work with python 2.7 ?

**Comment by JoaoFelipe** [comment]
It should work with both Python 2 and 3. Let me know if you find any issues

**Comment by vsraptor** [comment]
It seems it is accessible only for 3.0, how could I make it available also to 2.7

```
$ python -m apted -t {a{b}{c}} {a{b{d}}} -mv
/usr/bin/python: No module named apted

$ python3 -m apted -t {a{b}{c}} {a{b{d}}} -mv
distance:             2
runtime:              0.0001785755157470703
{a{b}{c}} -> {a{b{d}}}
{c} -> None
{b} -> {b{d}}
None -> {d}

```

**Comment by vsraptor** [comment]
this worked out :

`     # pip2 install apted`
